tests/data_context/store/test_database_store_backend.py

```diff
@@ -11,7 +11,7 @@
 pytestmark = pytest.mark.sqlalchemy_version_compatibility
 
 # module level markers
-pytestmark = [pytest.mark.postgres]
+pytestmark = [pytest.mark.postgresql]
```

Comment:
nit: we can remove brackets here. This is not blocking.

Reply:
done